KiCad.gitignore

```diff
@@ -5,9 +5,19 @@
 *.bak
 *.bck
 *.kicad_pcb-bak
+*~
+_autosave-*
+*.tmp
 
 # Netlist files (exported from Eeschema)
 *.net
 
 # Autorouter files (exported from Pcbnew)
 .dsn
+
+# Exported BOM files
+*.xml
+*.csv
+
+# Other system files
+.DS_Store
```

Comment:
This is an OSX specific rule, which we track in `Global/OSX.gitignore` alongside others. Can you drop this?